src/platform/functions/index.js

```diff
@@ -83,19 +83,42 @@ const scrapeCityEvents = async (city) => {
           parsedImgUrl = imgUrl.getAttribute('data-src');
         }
 
+        let lat = 0;
+        let lng = 0;
+        if (eventLoc) {
+          try {
+            const response = await fetch(`https://maps.googleapis.com/maps/api/geocode/json?address=${eventLoc.innerText}&key=AIzaSyCFm3jPZv8BgY7sXh5QS8X6WNpApmfD8OQ`);
```

Comment:
although we have security setup, we definitely do not want to expose this key, let's set it as a environment variable. Have a look at this guide https://firebase.google.com/docs/functions/local-emulator

Reply:
oh also remember to add `runtimeconfig.json` to gitignore